lib/configurators/aws/aws_test.go

```diff
@@ -39,15 +40,22 @@ import (
 	"github.com/gravitational/teleport/lib/cloud/mocks"
 	"github.com/gravitational/teleport/lib/config"
 	"github.com/gravitational/teleport/lib/configurators"
+	"github.com/gravitational/teleport/lib/service/servicecfg"
 	"github.com/gravitational/teleport/lib/services"
 )
 
 func TestAWSIAMDocuments(t *testing.T) {
 	userTarget, err := awslib.IdentityFromArn("arn:aws:iam::123456789012:user/example-user")
 	require.NoError(t, err)
 
-	roleARN := "arn:aws:iam::123456789012:role/example-role"
-	roleTarget, err := awslib.IdentityFromArn(roleARN)
+	internalAssumeRole1 := "arn:aws:iam::123456789012:role/internal-role-1"
+	internalAssumeRole2 := "arn:aws:iam::123456789012:role/internal-role-2"
+	internalAssumeRole3 := "arn:aws:iam::123456789012:role/internal-role-3"
+	externalAssumeRole1 := "arn:aws:iam::222222222222:role/external-role-1"
+	externalAssumeRole2 := "arn:aws:iam::222222222222:role/external-role-2"
+	externalAssumeRole3 := "arn:aws:iam::222222222222:role/external-role-3"
```

Comment:
should the tests be updated since no longer differentiate between internal vs external roles?

Reply:
yeah I guess there's no need for the distinction in tests anymore. will update

Reply:
0dded5313ee3750059eb4160aa2a497a14daa65c